code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -347,7 +347,7 @@
 	desc = "Consider this more of a \"Spell Bundle\". This artifact is NOT reccomended for weaklings. An ancient scroll that will teach you the art of Plasma Fist. With it's various combos you can knock people down in the area around you, light them on fire and finally perform the PLASMA FIST that will gib your target."
 	item_path = /obj/item/weapon/plasma_fist_scroll
 	log_name = "PF"
-	cost = 4  //VERY costly. Can only buy lvl1 magic missile/fireball/etc.
+	cost = 1  //It's a combination of kickass fighting moves, which will often require magic missile or some other disabling spell to work well.
```

Comment:
I suppose so its just not improperly reflecting it, although it obviously has a personal touch. If he lowered the cost to 1, and it still said "VERY COSTLY", looks a little silly to anyone reading.